common/data/src/main/java/org/thingsboard/server/common/data/notification/rule/trigger/NotificationRuleTriggerType.java

```diff
@@ -26,18 +28,15 @@ public enum NotificationRuleTriggerType {
     ALARM_ASSIGNMENT,
     DEVICE_ACTIVITY,
     RULE_ENGINE_COMPONENT_LIFECYCLE_EVENT,
-    NEW_PLATFORM_VERSION(false),
-    ENTITIES_LIMIT(false),
-    API_USAGE_LIMIT(false);
+    NEW_PLATFORM_VERSION(false, true),
+    ENTITIES_LIMIT(false, false),
```

Comment:
ENTITIES_LIMIT and API_USAGE_LIMIT are good candidates for deduplication, right?